spec/compiler/lexer/lexer_spec.cr

```diff
@@ -231,8 +231,8 @@ describe "Lexer" do
   it_lexes_number :i8, ["0i8", "0"]
 
   it_lexes_char "'a'", 'a'
-  it_lexes_char "'\\a'", 7.chr
-  it_lexes_char "'\\b'", 8.chr
+  it_lexes_char "'\\a'", '\u{7}'
```

Comment:
Why not `'\a'`?

Reply:
I did this because `p 7.chr` gives `'\u{7}'`

Reply:
@r00ster91 can you change the `\u{7}` to `\a` and also use `\b` in the following spec to keep it consistent with the rest of the suite.